app/Services/ImportService.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+/**
+ * Provide methods to import files.
+ */
+class FreshRSS_Import_Service {
+	/** @var string */
+	private $username;
+
+	/** @var FreshRSS_CategoryDAO */
+	private $catDAO;
+
+	/** @var FreshRSS_FeedDAO */
+	private $feedDAO;
+
+	/**
+	 * Initialize the service for the given user.
+	 *
+	 * @param string $username
+	 */
+	public function __construct($username) {
+		require_once(LIB_PATH . '/lib_opml.php');
+
+		$this->username = $username;
+		$this->catDAO = FreshRSS_Factory::createCategoryDao($username);
+		$this->feedDAO = FreshRSS_Factory::createFeedDao($username);
+	}
+
+	/**
```

Comment:
The whole code below comes verbatim from `app/Controllers/importExportController.php`